src/actions/security-chat.js

```diff
@@ -37,9 +39,31 @@ async function createMessage(newMessage) {
       body: JSON.stringify(newMessage),
       headers: { 'Content-Type': 'application/json' },
     });
-    const result = await response.json(response);
-    console.info('333', result);
-    return result;
+    return await response.json();
+  } catch (error) {
+    return error;
+  }
+}
+export function setCurrentMessage(messageId, text, item) {
+  return {
+    messageId,
+    editMessageText: text,
+    message: item,
+    type: EDIT_MESSAGE,
+  };
+}
+export async function fetchMessageById(messageId) {
```

Comment:
All the functions in actions folders should be action generators and `fetchMessageById` is not, this function is used only once and definitely was extracted in unnecessary decomposition. Please copy its contents and paste it in an action generator.

Reply:
I removed this function because I no longer need it. I`m already using data of current editing message from redux.

Reply:
If you removed it, why it's still present in the PR?

Reply:
It's weird. Locally and on my branch it is gone. I resolved conflicts and push changes.